### PR TITLE
add documentation for PUT /users/{user_id}/status

### PR DESCRIPTION
**Pull request by cpanato**
Implementation pr: https://github.com/mattermost/platform/pull/5928

### CODE REVIEW

File: v4/source/status.yaml [hunk 1]
```diff
@@ -16,3 +21,44 @@
...
+              user_id:
+                type: string
+              status:
+                type: string
+              manual:
+                type: boolean
+              last_activity_at:
+                type: integer
+              active_channel:
+                type: string
```
Comment (jwilander) [inline v4/source/status.yaml]: For the properties can we only include `status`? The endpoint only uses that one so we should make sure that the documentation doesn't imply you can update the other fields

File: v4/source/status.yaml [hunk 2]
```diff
@@ -16,3 +21,44 @@
           $ref: '#/responses/BadRequest'
         '401':
           $ref: '#/responses/Unauthorized'
+    put:
+      tags:
+        - status
+      summary: Update user status
+      description: |
+        Update user status.
```
Comment (jwilander) [inline v4/source/status.yaml]: Can we change this to `Manually set a user's status. When setting a user's status, the status will remain that value until set "online" again, which will return the status to being automatically updated based on user activity.`

File: v4/source/status.yaml (hunk 1 above)
Comment (cpanato) [inline v4/source/status.yaml]: done

File: v4/source/status.yaml (hunk 2 above)
Comment (cpanato) [inline v4/source/status.yaml]: done

**Pull request merged**